bin/kafka-run-class.sh

```diff
@@ -140,6 +140,11 @@ do
 done
 shopt -u nullglob
 
+if [ -z "$CLASSPATH" ] ; then
+  echo "Classpath is empty. Please build the project first e.g. by running './gradlew jarAll'"
```

Comment:
It seems like it would be better to recommend something like `./gradlew jar -PscalaVersion=$SCALA_VERSION` or something.